rllib/examples/centralized_critic_2.py

```diff
@@ -1,9 +1,9 @@
-"""An example of implementing a centralized critic by modifying the env.
+"""An example of implementing a centralized critic with ObservationFunction.
```

Comment:
user may ask, what is "ObservationFunction" given you don't use it here?

Reply:
It's the interface definition. You can extend it or not, I don't know how to document a function signature otherwise though